extensions/jdbc/jdbc-mysql/jdbc-mysql-runtime/src/main/java/io/quarkus/jdbc/mysql/runtime/graal/AbandonedConnectionCleanupThread_disable.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.jdbc.mysql.runtime.graal;
+
+import com.mysql.cj.MysqlConnection;
+import com.mysql.cj.jdbc.AbandonedConnectionCleanupThread;
+import com.mysql.cj.protocol.NetworkResources;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+@Substitute
+@TargetClass(AbandonedConnectionCleanupThread.class)
+final public class AbandonedConnectionCleanupThread_disable {
+
+    @Substitute
+    protected static void trackConnection(MysqlConnection conn, NetworkResources io) {
```

Comment:
I think having a comment explaining why the substitution would be nice. It might be clear now but it won't be two years from now :).

Reply:
This is to disable connection clean up thread which launches a thread on a static block. Graal VM is not happy about that. Guess the cache extension that @gwenneg was working had the same issue with Caffeine lib.

Reply:
I wonder if the right fix for now would be to initialize the class at runtime instead of initializing it at runtime.

Could you try if it works?